### DISCUSSION TITLE
Scenarios and Sensitivities for the EURO2024

### DISCUSSION THREAD

**Discussion by hannesfelipe** [CSEI-EU/Nord_H2ub#41] [General]
This should be the base case, scenarios and sensitivities we want to calculate for the EURO2024:

1. Base case: 

- capacity is based on public information for electrolysis and PV plant
- further unit capacity results from maximal flows for the units
- storage capacity either max state or reasonable capacity based on public information
- cost data from 2020 (technology catalogue by DEA)
- power prices from 2019 ("normal" year before COVID, Russian Invasion and energy crisis)
- demand either not fixed (endogenous flexibility) or via trucks (exogenous given time)
- electrolysis with piecewise linear efficiency representation

2. Scenarios

- electrolysis with constant efficiency
- PV power cannot be sold on the wholesale market
- power supply only by PV plant
- one run with a power price curve of a year with a crisis (e.g. 2022)

3. Sensitivities

- waste heat price (0-100% of the possible price for third-party district heating supply)
- wholesale power price level variation
- wholesale power price variability variation

LCOE calculation

- one with and one without considering the PV power revenue on the wholesale market
- provide a representation that shows the contribution of the different cost 

@danajhen please add.

**Comment by hannesfelipe** [comment]
One more scenario: instead of using the PV power at a cost of 0, assuming a PPA and a price for the PV power. Thus, the plant uses all available PV input, if possible but has to pay for it.

**Comment by danajhen** [comment]
Sensitivities for different demand levels

**Comment by hannesfelipe** [comment]
- [x] Base Case

- [x] Scenario with constant efficiency
- [x] Power supply only by PV plant (no fix demand in this case)
- [x] PV power cannot be sold on the wholesale market (instead of a unbalanced pc power node the cost for export is 1000000.0 - to avoid infeasible solutions) 
- [x] PV Power at price of X (PPA)
- [x] run with a power price curve of a year with a crisis (e.g. 2022)

- [x] Wast heat price sensitivities [ ] 0% [ ] 10% [ ] 20% [X] 30% [X] 40%  [BASE] 50% [X] 60% [X] 70% [ ] 80% [ ] 90% [ ] 100% of price cap
- [x] power price level [ ] -50% [ ] -40% [ ] -30% [ ] -20%  [X] -10% [X] +10% [ ] +20% [ ] +30% [ ] +40% [ ] +50% 
- [x] power price variability [ ] -50% [ ] -40% [ ] -30% [ ] -20%  [X] -10% [X] +10% [ ] +20% [ ] +30% [ ] +40% [ ] +50% 
- [x] Demand level [ ] -50% [ ] -40% [ ] -30% [ ] -20%  [X] -10% [X] +10% [X] +20% [ ] +30% [ ] +40% [ ] +50%

**Comment by hannesfelipe** [comment]
finally, we just did sensitivity for +/- 10% as we did not have more time.
